apps/src/templates/teacherDashboard/TeacherDashboard.jsx

```diff
@@ -179,6 +187,19 @@ function TeacherDashboard({
             <SectionAssessments sectionName={sectionName} />
           )}
         </Route>
+        {teacherNavV2Enabled && (
+          <Route
+            path={TeacherDashboardPath.navTestV2}
+            component={props => (
+              <div className={dashboardStyles.pageContainer}>
+                <TeacherNavigationBar />
+                <div className={dashboardStyles.content}>
+                  <SectionProgressSelector />
```

Comment:
This SectionProgressSelector is only intended to help visualize what it would look like in the future.  The focus of this PR is on the sidebar, but I thought the inclusion of this component here is a more realistic way to see what the page will look like.